components/notion/actions/update-page/update-page.mjs

```diff
@@ -15,7 +15,7 @@ export default {
   key: "notion-update-page",
   name: "Update Page",
   description: "Updates page property values for the specified page. Properties that are not set will remain unchanged. To append page content, use the *append block* action. [See the docs](https://developers.notion.com/reference/patch-page)",
-  version: "0.3.0",
+  version: "0.3.7",
```

Comment:
```suggestion
  version: "0.3.7",
```

Reply:
Added a comma at the end of the line to maintain consistency with other lines in the CSV.